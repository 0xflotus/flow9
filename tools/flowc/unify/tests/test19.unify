import maybe;

α1 ={union Maybe<α10005> .. *};

α4 = Some<α10006>;

α10006 = { union Maybe<α10005> .. *};

α10005 = {None<α10003> .. *};

α4 c= α1;
