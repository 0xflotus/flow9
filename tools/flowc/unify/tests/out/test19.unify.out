α1={Maybe<{Maybe<α10005> .. *}> .. *}
α4=Some<{Maybe<{Maybe<α10005> .. *}> .. *}>
α10003={None<{None<α10003> .. *}> .. *}
α10005={Maybe<{Maybe<α10005> .. *}> .. *}
α10006={Maybe<{Maybe<α10005> .. *}> .. *}
